app/views/layouts/_header.html.erb

```diff
@@ -36,14 +36,14 @@
         <li class="nav-item dropdown">
           <a class="nav-link request-link" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" title="Friend Requests" aria-haspopup="true" aria-expanded="false">
             <%= fa_icon "users", class: "user-icon nav-icon" %> 
-            <% if current_user.pending_requests_from_users.count.positive? %>
-              <span class="badge badge-light request-count"><%= current_user.pending_requests_from_users.count %></span>
+            <% if current_user.incoming_pending_friends.count.positive? %>
+              <span class="badge badge-light request-count"><%= current_user.incoming_pending_friends.count %></span>
```

Comment:
**[REQUIRED]**
* Let's separate code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).